components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css

```diff
@@ -125,3 +125,9 @@
 .filter-match-count {
   margin-left: 0.5em
 }
+input.suggestUsersAndGroups {
```

Comment:
This doesn't really belong here, move it to `PhenoTips.Widgets`, in the SSX object named `Suggest workgroups`, which should be renamed to `Suggest users, groups and workgroups`.